cmd/podman/containers/wait.go

```diff
@@ -76,6 +74,10 @@ func wait(cmd *cobra.Command, args []string) error {
 		return errors.New("interval must be greater then 0")
 	}
 
+	if !waitOptions.Latest && len(args) == 0 {
+		return errors.Errorf("%q requires a name, id, or the \"--latest\" flag", cmd.CommandPath())
+	}
+
```

Comment:
Arg, something's not right on my VM and I can't compile this (or anything else at the moment).  Will a command like this be accepted?

`podman wait --latest ctrid1 ctrid2`

Reply:
```
$ ./bin/podman wait --latest foo1 foo2
0
```
(Looks like "-l" overrode the explicit IDs, but I can't tell)

Reply:
It will be accepted but it will only use the latest container and ignore the arguments in this case. Not sure if this is should be the correct behavior.

Reply:
Yeah, that's broken:
```console
$ ./bin/podman wait -l foo3 slartibartfast this-container-has-never-existed
2
```

Reply:
I think throwing an error when both latest and args are used would be correct. Right?!

Reply:
Correct. See, e.g.,
```console
$ ./bin/podman inspect -l foo
Error: latest and containers are not allowed

$ ./bin/podman rm -l foo
Error: no arguments are needed with --latest

$ ./bin/podman top -l foo
PID   USER     TIME  COMMAND
    1 root      0:00 sleep 90
(oops! Time to file a bug report!)
```